src/main/java/ua/danit/final_project/entities/Role.java

```diff
@@ -0,0 +1,45 @@
+package ua.danit.final_project.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+
+@Entity
+@Table(name = "role")
+@Data
+public class Role implements Serializable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @JsonIgnore
+  private Long id;
+
+  @Column(name = "r_name", unique = true, nullable = false)
+  private String name;
+
+  @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+  @JoinTable(
+          name = "role_permission",
+          joinColumns = {@JoinColumn(name = "r_id")},
+          inverseJoinColumns = {@JoinColumn(name = "p_id")})
+  public List<Permission> permissions;
+
+  @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+  @JoinTable(name = "user_role",
+          joinColumns = {@JoinColumn(name = "r_id")},
+          inverseJoinColumns = {@JoinColumn(name = "u_id")})
+  public List<User> users;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "users" private or transient. ([squid:S1948](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1948))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=80&issues=AWVMbp7blRPS_ea-WxrR&open=AWVMbp7blRPS_ea-WxrR)

[ISSUE_KEY]: AWVMbp7blRPS_ea-WxrR